Localization/Translations/MinimapIcon/Townsfolk.lua

```diff
@@ -252,19 +252,19 @@ local townsfolkLocales = {
         ["zhCN"] = "商人",
         ["zhTW"] = "交易員",
         ["esES"] = "Objetos comerciables",
-        ["frFR"] = "Fournitures",
+        ["frFR"] = "Fournitures d'artisanat",
     },
     ["Vendor"] = {
-        ["ptBR"] = "Vendedor",
+        ["ptBR"] = "Comerciante",
         ["ruRU"] = "Торговец",
-        ["deDE"] = "Händler",
+        ["deDE"] = "Verkäufer",
```

Comment:
"Händler" fits better for this translation.

```suggestion
        ["deDE"] = "Händler",
```

Reply:
@BreakBB Actually if you look at the words used in the german localization, it's Verkäufer

See
https://de.classic.wowhead.com/npc=3323/horthus
https://de.classic.wowhead.com/npc=1257/keldric-boucher
https://de.classic.wowhead.com/npc=1285/thurman-mullby
https://de.classic.wowhead.com/npc=4585/ezekiel-graves
https://de.classic.wowhead.com/npc=3369/gotri